drivers/led/lp5569.c

```diff
@@ -156,6 +159,8 @@ static const struct led_driver_api lp5569_led_api = {
 #define LP5569_DEFINE(id)						\
 	static const struct lp5569_config lp5569_config_##id = {	\
 		.bus = I2C_DT_SPEC_INST_GET(id),			\
+		.cp_mode = DT_PROP_OR(DT_DRV_INST(id),			\
+				      charge_pump_mode, 0),		\
```

Comment:
This will ensure that `cp_mode` holds a supported value (which might not be the case with current code).